src/core/src/type/element_type.cpp

```diff
@@ -71,6 +71,10 @@ inline TypeInfo get_type_info(ov::element::Type_t type) {
         return {64, false, false, false, "uint64_t", "u64"};
     case ov::element::Type_t::nf4:
         return {4, false, false, true, "nfloat4", "nf4"};
+    case ov::element::Type_t::f8e4m3:
+        return {8, true, true, true, "f8e4m3", "f8e4m3"};
+    case ov::element::Type_t::f8e5m2:
+        return {8, true, true, true, "f8e5m2", "f8e5m2"};
```

Comment:
@AlexKoff88, @andreyanufr Could you advice, should the `is_quantized` flag be set to `true` or `false` for f8 types?
https://github.com/openvinotoolkit/openvino/blob/d1f89dead52a11b08a8dc76a5c670e8328546b7f/src/core/src/type/element_type.cpp#L15-L22

Reply:
I am not aware of what this flag affects really in runtime. @slyalin, @eshoguli, @dmitry-gorokhov, can anyone help here?

Reply:
I've check it looks like is not used often. I found one usage in GPU plugin in function
```cpp
    static bool is_i8_u8(data_types data_type) {
        auto et = ov::element::Type(data_type);
        return et.is_quantized() && et.bitwidth() == 8;
    }
```
This will return true result for unwonted types but it looks like local implementation issue.

Reply:
@vladimir-paramuzov, @p-durandin to comment on this.

Reply:
@eshoguli, maybe you have details about the mentioned flag?

Reply:
@AlexKoff88 I can't comment on the original intent of this flag. Usage in GPU plugin mentioned above doesn't expect fp8 type, but we don't support fp8 at all for now, so any value won't cause a problem. We can change this check to `et.is_integral_number() && et.bitwidth() == 8;` or something like that later if needed. 

As for original question, I'd say that flag should be set to `true`, but we can also consider removal of this property at all if it's not really widely used and definition is somewhat blurry